supportingcharts/README.md

```diff
@@ -64,94 +65,132 @@ spec:
+
+```bash
+kubectl adm policy add-scc-to-user anyuid -nsample -z default
+```
+
+Now, install this chart with `helm`, where we assume you working in the same directory as this README file, i.e., `supportingcharts`:
+
+```bash
+helm install extendedcassandrachart
+```
+
 After the chart is installed you should see the following:
-* cassandra stateful set and corresponding pods running cassandra itself
-* cassandra-reaper deployment with corresponding pod running cassandra
-* cassandra service providing access to Cassandra
-     
-Note: There still can be a problem with cassandra-reaper pod. Although most of the time reaper-db is created by a script, sometimes it does not. If a reaper fails to start properly go to cassandra-0 pod terminal and run following commands:````     
-````
+
+* `cassandra` stateful set and corresponding pods running Cassandra itself
+* `cassandra-reaper` deployment with the corresponding pod running Cassandra
+* `cassandra` service providing access to Cassandra
+
+Note that sometimes the `cassandra-reaper` pod fails to start correctly; sometimes the `reaper-db` is not successfully created by an embedded script. When this happens, open the `cassandra-0` pod's terminal and run following commands:
+
+```bash
 # cqlsh
 Connected to sample at 127.0.0.1:9042.
 [cqlsh 5.0.1 | Cassandra 3.11.1 | CQL spec 3.4.4 | Native protocol v4]
 Use HELP for help.
 cqlsh> CREATE KEYSPACE IF NOT EXISTS reaper_db WITH replication = {'class': 'NetworkTopologyStrategy', 'DC1': 1}; DESCRIBE keyspaces;
-     
+
 reaper_db  system_schema  system_auth  system  system_distributed  system_traces
-     
+
 cqlsh> exit;
-#
-````     
+```
+
+Programmatic access to Cassandra is through the service `cassandra` using host `cassandra.sample.svc` and port `9042`.
+
+### Using the Cassandra Chart (OpenShift)
+
+The simpler [`cassandrachart` helm chart](./cassandrachart) starts Cassandra without the reaper and works better on OpenShift.
 
-Programmatic access to cluster (inside openshift cluster) by client in this case is through the service  `cassandra using` host `cassandra.sample.svc`  and port `9042`
-     
-After experimenting with this helm chart, it turns out to be unstable on Openshift (stable on GKE), so I switched to a simpler [helm chart](../../supportingcharts/cassandrachart). This one starts Cassandra without reaper and works out of the box.
+```bash
+helm install cassandrachart
+```
 
-## NFS installation
-NFS installation is based on the following [helm chart](../../supportingcharts/nfschart). This chart requires privilege user (yes its a security risk, but okay for sample application). This can be achieved using the following command 
-````     
+Programmatic access to Cassandra is through the service `cassandra` using host `cassandra.sample.svc` and port `9042`.
+
+## NFS Installation
+
```

Comment:
Base on our discussion, should we talk about NFS or steer people towards GlusterFS

Reply:
Added callout "Note" that we recommend GlusterFS and pointed reader to docs section about it.